src/core/src/service_interfaces/StatusHandler.py

```diff
@@ -551,23 +568,19 @@ def load_status_file_components(self, initial_load=False):
 
         self.composite_logger.log_debug("Loading status file components [InitialLoad={0}].".format(str(initial_load)))
 
-        # Verify the status file exists - if not, reset status file
-        if not os.path.exists(self.status_file_path) and initial_load:
+        # Verify the complete status file exists - if not, reset complete status file
+        if os.path.isdir(self.complete_status_file_path):
```

Comment:
self.complete_status_file_path is set in ExecutionConfig, to a *.complete.status file, so when will this condition be true?